pkg/kubectl/describe/versioned/describe.go

```diff
@@ -881,8 +881,8 @@ func printProjectedVolumeSource(projected *corev1.ProjectedVolumeSource, w Prefi
 				"    ConfigMapOptional:\t%v\n",
 				source.ConfigMap.Name, source.ConfigMap.Optional)
 		} else if source.ServiceAccountToken != nil {
-			w.Write(LEVEL_2, "TokenExpirationSeconds:\t%v\n",
-				source.ServiceAccountToken.ExpirationSeconds)
+			w.Write(LEVEL_2, "TokenExpirationSeconds:\t%d\n",
+				*source.ServiceAccountToken.ExpirationSeconds)
```

Comment:
Do we need to worry about ExpirationSeconds being nil?

Reply:
[https://github.com/kubernetes/kubernetes/blob/3156df3e806cb8bf468845b37eec939dee52a1d8/pkg/apis/core/v1/defaults.go#L239-L241](https://github.com/kubernetes/kubernetes/blob/3156df3e806cb8bf468845b37eec939dee52a1d8/pkg/apis/core/v1/defaults.go#L239-L241)

@justinsb if it's properly decoded from the apimachinery stack, no.